### PR TITLE
Disable resize

### PR DESCRIPTION
**Pull request by joechrz**
surfaces an option to disable column resizing similar to how disableReorder works

### CODE REVIEW

File: src/modules/core/index.ts [hunk 1]
```diff
@@ -293,7 +294,10 @@ export function create(opts: IGridOpts = {}): Grid {
 
   grid.pixelScrollModel = createPixelScrollModel(grid);
   grid.showHiddenCols = createShowHiddenCols(grid);
-  grid.colResize = createColResize(grid);
+
+  if (!(opts.col && opts.col.disableResize)) {
```
Comment (scamden) [inline src/modules/core/index.ts]: `if(opts.col && !opts.col.disableResize)` right? otherwise it disables when there's no column opts provided

Comment (scamden) [reply src/modules/core/index.ts]: I guess the current one is wrong too lol, unless the default is to disable..

Comment (joechrz) [reply src/modules/core/index.ts]: i think this is right actually- your suggestion would only enable the resizing logic if `opts.col` was specified.

but i'd be down to switch up the conditional to make it easier to read:

```typescript
if (!opts.col || !opts.col.disableResize) {
  grid.colResize = createColResize(grid);
}
```

Comment (scamden) [reply src/modules/core/index.ts]: oh right duh, flipping the disable, double negatives lol, I like the `!opt.col || !opts.col.disableResize` version personally, but this is fine

**Pull request merged**